cirq-ionq/cirq_ionq/job_test.py

```diff
@@ -100,6 +100,19 @@ def test_job_results_qpu_endianness():
     results = job.results()
     assert results == ionq.QPUResult({0: 600, 2: 400}, 2, measurement_dict={})
 
```

Comment:
black wants an extra space here

Reply:
thanks! I still gotta get my local black/flynt etc setup the same as in CI, I think :)